plugin/adsk/scripts/mayaUsdTranslatorExport.mel

```diff
@@ -15,7 +15,12 @@
 // limitations under the License.
 //
 
+global string $mayaUsdTranslatorExport_SectionNames[];
+
 proc mayaUsdTranslatorExport_SetCheckbox(string $arg, int $enable, string $widget) {
+    if (`checkBoxGrp -q -exists $widget` == 0)

```

Comment:
There are a lot of changes in this files to cope with the fact that some UI are not always created.

Reply:
I don't know if there is a side-effect, to adding `-q`, but its not needed since the `-exists` flag is a **create** flag (not a query flag).

Reply:
I think I agree with Sean here.  The docs (including the docs to the very general "control" command) say that exists is a create flag.  I'd drop the -q, even though in current code it doesn't seem to matter, just to respect the spec and be future-proof.